doc/overview.rst

```diff
@@ -2394,15 +2448,15 @@ configuration file.
 -  Model: ``CreateModel``, ``UpdateModel``
 -  Model Package: ``CreateModelPackage``, ``UpdateModelPackage``
 -  Processing Jobs: ``CreateProcessingJob``, ``UpdateProcessingJob``
--  Training Job: ``CreateTrainingJob``, ``UpdateTrainingJob``
```

Comment:
We would like to keep the UpdateTrainingJob in the list of API's since we support it.

Reply:
@akrishna1995 Identified this as an unsupported API. If that isn't the case, I'll add it back it!

Reply:
Sorry my bad - we initially did not support the resource_config injection, but later provided support to profiler object as a PR comment

Reply:
Done